clock-mcu/src/firmware/config.h

```diff
@@ -18,12 +18,13 @@
 struct ConfigData {
   bool twelveHMode; // True = 12h; False = 24h
   bool dayNightIndicator; // True = enable; False = disable
+  int  blink; // 0 = none; 1 only during the day; 2 always
```

Comment:
I would think that 1 is the normal on, and 2 would be the configured only during the day, but curious what others think?

Reply:
I'm currently preparing a setting card where this value will be mapped on a slider. So the value is increasing for more *blinking time* during the day (0 no blink, 1 blink during day, 2 always blink).

Reply:
I agree with 1 being normal on and 2 being a special setting